.phpunit.result.cache

```diff
@@ -0,0 +1 @@
testIsCommentedDoctrineType with data set "binary"";d:0;s:49:"testIsCommentedDoctrineType with data set "float"";d:0;s:51:"testIsCommentedDoctrineType with data set "decimal"";d:0;s:58:"testIsCommentedDoctrineType with data set "time_immutable"";d:0;s:48:"testIsCommentedDoctrineType with data set "time"";d:0;s:58:"testIsCommentedDoctrineType with data set "date_immutable"";d:0;s:48:"testIsCommentedDoctrineType with data set "date"";d:0;s:64:"testIsCommentedDoctrineType with data set "datetimetz_immutable"";d:0;s:62:"testIsCommentedDoctrineType with data set "datetime_immutable"";d:0;s:56:"testIsCommentedDoctrineType with data set "simple_array"";d:0;s:52:"testIsCommentedDoctrineType with data set "datetime"";d:0;s:48:"testIsCommentedDoctrineType with data set "text"";d:0;s:50:"testIsCommentedDoctrineType with data set "string"";d:0;s:50:"testIsCommentedDoctrineType with data set "bigint"";d:0;s:52:"testIsCommentedDoctrineType with data set "smallint"";d:0;s:51:"testIsCommentedDoctrineType with data set "integer"";d:0;s:51:"testIsCommentedDoctrineType with data set "boolean"";d:0;s:50:"testIsCommentedDoctrineType with data set "object"";d:0;s:48:"testIsCommentedDoctrineType with data set "json"";d:0;s:54:"testIsCommentedDoctrineType with data set "json_array"";d:0;s:56:"testIsCommentedDoctrineType with data set "dateinterval"";d:0;s:51:"testRegistersCommentedDoctrineMappingTypeImplicitly";d:0;s:38:"testRegisterUnknownDoctrineMappingType";d:0;s:31:"testRegisterDoctrineMappingType";d:0;s:33:"testGetUnknownDoctrineMappingType";d:0;s:44:"testGetInvalidForeignKeyReferentialActionSQL";d:0;s:58:"testReturnsForeignKeyReferentialActionSQL with data set #0";d:0;s:58:"testReturnsForeignKeyReferentialActionSQL with data set #1";d:0;s:58:"testReturnsForeignKeyReferentialActionSQL with data set #2";d:0;s:58:"testReturnsForeignKeyReferentialActionSQL with data set #3";d:0;s:58:"testReturnsForeignKeyReferentialActionSQL with data set #4";d:0;s:58:"testReturnsForeignKeyReferentialActionSQL with data set #5";d:0;s:25:"testQuoteSingleIdentifier";d:0;s:19:"testQuoteIdentifier";d:0;s:33:"testColumnCollationDeclarationSQL";d:0;s:61:"testDoesNotPropagateForeignKeyCreationForNonSupportingEngines";d:0.001;s:27:"testSupportsColumnCollation";d:0;s:44:"testListTableForeignKeysSQLEvaluatesDatabase";d:0;s:43:"testQuotesDatabaseNameInListTableColumnsSQL";d:0;s:40:"testQuotesTableNameInListTableColumnsSQL";d:0;s:47:"testQuotesDatabaseNameInListTableForeignKeysSQL";d:0;s:44:"testQuotesTableNameInListTableForeignKeysSQL";d:0;s:36:"testQuotesDatabaseNameInListViewsSQL";d:0;s:43:"testQuotesDatabaseNameInListTableIndexesSQL";d:0;s:40:"testQuotesTableNameInListTableIndexesSQL";d:0;s:33:"testReturnsGuidTypeDeclarationSQL";d:0;s:58:"testDoesNotPropagateDefaultValuesForUnsupportedColumnTypes";d:0.001;s:63:"testDoesNotPropagateForeignKeyAlterationForNonSupportingEngines";d:0.001;s:35:"testZeroOffsetWithoutLimitIsIgnored";d:0;s:45:"testIdempotentDataConversion with data set #0";d:0.179;}}}
```

Comment:
Should this really be under version control?

Reply:
No, I accidentally committed it when I tried using new PHPUnit flags like `--cache-result` combined with `--order-by`. Does anyone have experience using them?

Reply:
I read about them the other day, but did not try yet… but it sounds cool!